instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/instrumentation.rb

```diff
@@ -30,6 +30,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         option :peer_service, default: nil, validate: :string
         option :enable_sql_obfuscation, default: false, validate: :boolean
         option :db_statement, default: :include, validate: %I[omit include obfuscate]
+        option :span_name, default: :statement_type, validate: %I[statement_type db_name db_operation_and_name]
```

Comment:
not exactly the most descriptive configuration option but i don't really have any better ideas